Skills.md

```diff
@@ -4,22 +4,22 @@ https://api.careerbuilder.com/core/tagging/skills
 
 This API supports the HTTP/GET and HTTP/POST methods.  
 
-This API currently supports versions 4.1 and 4.2. Version is passed in the Accept header as the parameter version (Ex. Accept:application/json;version=4.1). 4.2 contains the latest taxonomy and incorporates feedback received about specific terms since the release of V4/4.1.
+This API currently supports versions 4.1, 4.2, and 5.0. Version is passed in the Accept header as the parameter version (Ex. Accept:application/json;version=4.1). 5.0 contains the latest taxonomy and incorporates feedback received about specific terms since the release of V4/4.1 and V4.2.
 
 **This API is also available as a locally-runnable JAR. Clients with JVM-based applications may find this useful to avoid the overhead of repeated network calls. More information can be found [here](https://github.com/cbdr/SkillsExtractor).**
 
-| Parameter (4.1/4.2) | Required | Description |
+| Parameter (4.1/4.2/5.0) | Required | Description |
 |----------------|----------------|-----------------|
-| version        | Yes | Passed in via the Accept header. Possible values are "4.1" or "4.2". | 
+| version        | Yes | Passed in via the Accept header. Possible values are "4.1", "4.2", or "5.0". | 
 | content        | Yes | A string containing the content to be tagged |
 | language            | No | A string determining the language (total 22 languages supported) in which the input text is written. Default value is en. Note that the input parameter passing to language should be the language id. |
 | threshold      | No | A double value between 0 and 1 controlling minimum relevancy scores for skill recognition. Higher values will more tightly restrict the returned skill tags. Default is 0.5. A threshold of 0 means all seed skill phrases recognized by exact string matching will be returned. Note that this parameter is only supported for inputs in English. |
-| auto_thres     | No | A boolean value indicating whether automatic thresholding is desired. If enabled, then when input text contains 150 or fewer words, the threshold parameter will be overwritten to 0 and all confidence values will be overwritten to 1.0&#42;. Default is true.  Note that this parameter is only supported for inputs in en, fr, and de. |
+| auto_thres     | No | A boolean value indicating whether automatic thresholding is desired. If enabled, then when input text contains 150 or fewer words , the threshold parameter will be overwritten to 0 and all confidence values will be overwritten to 1.0&#42;. Default is true.  Note that this parameter is only supported for inputs in en, fr, and de. |

```

Comment:
I think a stray space got in somehow .

Reply:
yeah, one after `words` and there are two spaces before `Note`